src/rust/engine/workunit_store/src/metrics.rs

```diff
@@ -51,6 +51,9 @@ pub enum Metric {
   RemoteStoreMissingDigest,
   /// Number of times that we backtracked due to missing digests.
   BacktrackAttempts,
+  DockerExecutionRequests,
+  DockerExecutionSuccess,
```

Comment:
We call this Successes for RemoteCacheWrite, but singular for RemoteExecution. I think probably plural is better as we use plural generally